cloudinit/config/schema.py

```diff
@@ -371,7 +424,15 @@ def handle_schema_args(name, args):
         else:
             print("Valid cloud-config file {0}".format(args.config_file))
     if args.doc:
+        schema_ids = [subschema['id'] for subschema in full_schema['allOf']]
+        schema_ids += ['all']
+        if args.module and args.module not in ['all'] + schema_ids:
```

Comment:
I see.  Could I suggest that we convert --module to an input value?

--doc  # no args prints all module schema docs
--doc write_files   # prints write_file schema docs
--doc write_files runcmd   # prints two module schema docs ?

Reply:
I've changed it to allow --docs all      and or any number of individual cc_<modulename> params:
-- docs all # print everything
--docs cc_runcmd cc_resizefs cc_bootcmd